workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -82,11 +83,26 @@ spec:
         - name: red
           model: e1000 # expose e1000 NIC to the guest
           bridge: {} # connect through a bridge
+          ports:
+           - name: http
+             port: 80
   networks:
   - name: red
     pod: {}
 ```
 
+### Ports
+
+Ports will be configured as container ports for the virt-launcher pod.
```

Comment:
This is user guide. We should not refer to internal implementation details like virt-launcher pod here. Instead, please describe what the intent of the field is.